### ISSUE TITLE
Make the GitHub repository a template repository

### ISSUE THREAD

**Issue by lslezak** [cockpit-project/starter-kit#390]
GitHub allows marking a repository as a template repository. This easily allows using the template as a starting point for your new project.

Forks refer to the original repository and have a full history. Repositories created from a template are standalone repositories (not linked to the original source) and have the current state as the initial commit (no history). I think that the template approach works better for the purpose of this repository.

See the GitHub documentation: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/creating-a-template-repository

**Comment by martinpitt** [comment]
I think that makes sense. Honestly I don't mind the history much, as that might be insightful to understand why the project files are the way they are. But more importantly, fork behaviour is weird -- e.g. https://github.com/skobyda/cockpit-certificates is a fork from https://github.com/cockpit-project/starter-kit , and if you want to fork c-certificates it still "counts" as a fork of the original starter-kit. I avoided that problem with https://github.com/martinpitt/performance-graphs/ by not forking starter-kit on GitHub, but forking it locally and just pushing it to a new project. But a template project may avoid that problem also.

@marusak, I'd like a second opinion about this. WDYT?

**Comment by marusak** [comment]
I think it makes sense.
One issue I remember I had was when I tried to create fork for https://github.com/skobyda/cockpit-certificates it told me I already have fork of it. I of course did not have, but it meant that I have fork of starter-kit. So I ended up just managing branches in my starter-kit fork to work with certificates. It is a bit weird experience so if template repository makes this behave like a normal project, I am in favor.

> Honestly I don't mind the history much, as that might be insightful to understand why the project files are the way they are.

Yeah, I don't think that history is harmful but also I don't think very useful. When author wants to know why this file is the way it is, still can jump into starter-kit and check history here. And having clean history makes it feel more like proper project and not just "fork I edited" :)
